include/gf/Shape.h

```diff
@@ -87,16 +87,35 @@ inline namespace v1 {
      * use it, the behavior is undefined.
      *
      * If `resetRect` is true, the texture rect property of
-     * the shape is automatically adjusted to the size of the new
+     * the sprite is automatically adjusted to the size of the new
```

Comment:
copy-paste is bad :P

Reply:
This has been fixed in b70b0b728f7fc5b8ddc17e42ede28a55c978c1f8